### PR TITLE
Fix missing myVersion prop in trade checklist update message

### PR DESCRIPTION
**Pull request by SamTremko**

### CODE REVIEW

**Comment by SamTremko** [comment]
* **#991** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/991?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#984** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/984?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#983** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/983?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#982** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/982?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @SamTremko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/mobile/src/state/chat/atoms/sendTradeChecklistUpdateActionAtom.ts [hunk 1]
```diff
@@ -64,6 +65,7 @@ export default function createSubmitChecklistUpdateActionAtom(
             },
             time: unixMillisecondsNow(),
             uuid: generateChatMessageId(),
+            myVersion: version,
```
Comment (kaladivo) [inline apps/mobile/src/state/chat/atoms/sendTradeChecklistUpdateActionAtom.ts:68]: Good catch. It's not necessary, but it's good to have this here.

**Review by kaladivo — APPROVED** [review]

File: apps/mobile/src/state/chat/atoms/sendTradeChecklistUpdateActionAtom.ts (hunk 1 above)
Comment (SamTremko) [reply apps/mobile/src/state/chat/atoms/sendTradeChecklistUpdateActionAtom.ts:68]: Actually without this parameter, if one side approved the trade and right away sent trade checklist message (date, price, place of meet or other) and the other side received the message on the background, not with the chat detail open, it does not display the trade checklist messages after opening the chat detail, as the app did not know the version from "last message received". It displayed the messages once received message that contained this attribute (lets say normal message). It was the bug that slush reported, that the other side did not receive his trade checklist updates

**Comment by SamTremko** [comment]
### Merge activity

* **Apr 19, 5:10 AM EDT**: @SamTremko merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/983).

**Pull request merged**